class_photo/face.py

```diff
@@ -7,20 +7,20 @@
 def crop(imgs):
```

Comment:
A function description would be good for the documentation. Like for this function, it can be 
```py
'''
This function takes an image list and crops them to the size 200*200.
'''
```